uffizzi/docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: postgraphile
+    port: 5000
+
+services:
+  db:
+    restart: always
+    image: "${DB_IMAGE}"
+    environment:
+        # Add your credentials as GitHub Actions Secrets, then use the commented syntax below:
+        POSTGRES_USER: postgres # "${PGUSER}"
+        POSTGRES_PASSWORD: postgres # "${PGPASSWORD}"
+        POSTGRES_DB: forum_example # "${POSTGRESDB}"
+    ports:
+        - "5432:5432"
+
+  postgraphile:
+    image: "${APP_IMAGE}"
+    depends_on:
+      - db
+    ports:
```

Comment:
I don't think `ports`, `depends_on`, or `restart` work on Uffizzi?

Reply:
OK, I will delete it.